### ISSUE TITLE
Major refactoring on Citizen-integrator-track

### ISSUE THREAD

**Issue by hodrigohamalho** [RedHatWorkshops/dayinthelife-integration#377]
Major refactoring focusing on 3Scale and Fuse Online.

The motivation about it is that most part of the audience is in their first experience with Red hat Integration, so we need to make it smoother exploring more the core of Red Hat Integration (3Scale and Fuse on that case).

Proposed agenda:
- Design Locations API (same)
- Mock the API (same)
- Implement the API on Fuse Online (new)
-- List and Create methods
- Import this API on 3Scale (new)
-- Do all the mapping and use user-key authentication 
- Register on the developer portal the locations app (same)
- Deploy the Locations web app 
- User starts to create the API methods on 3Scale to see they discriminated on Analytics dashboard (new)
- Create a different plan and enable billing (new)
-- Register a new application on that plan (new)
- Bonus Lab (Secure your application with OpenID)

**Comment by weimeilin79** [comment]
Are you suggesting removing the developer track? And focus just on Integrator Track?

**Comment by hodrigohamalho** [comment]
@weimeilin79 no. But my suggestions are specific for improvements on the Integrator Track, maybe we should open another issue to consolidate some to Dev Track.

**Comment by GrooveCS** [comment]
@weimeilin79 I vote to keep them separate. Change the integrator track to engineering or administrative path is a possibility. I believe they have distinct interests different from developers.

**Comment by GrooveCS** [comment]
Propose for discussion the following be considered for web service gateway, proxy, and management. 

1. OpenAPI, SOAP/WSDL and/or OData/GraphQL definition be used as an example use-case
1. Intranet & Extranet example use-case
1. Aggregation of public services use-case
